x-pack/plugins/security_solution/public/management/components/endpoint_responder/components/offline_callout.tsx

```diff
@@ -32,16 +33,20 @@ export const OfflineCallout = memo<OfflineCalloutProps>(({ agentType, endpointId
   const sentinelOneManualHostActionsEnabled = useIsExperimentalFeatureEnabled(
     'sentinelOneManualHostActionsEnabled'
   );
+  const crowdstrikeManualHostActionsEnabled = useIsExperimentalFeatureEnabled(
+    'responseActionsCrowdstrikeManualHostIsolationEnabled'
+  );
 
   const { data: endpointDetails } = useGetEndpointDetails(endpointId, {
     refetchInterval: 10000,
     enabled: isEndpointAgent,
   });
 
   const { data } = getAgentStatus([endpointId], agentType, {
-    enabled: sentinelOneManualHostActionsEnabled && isSentinelOneAgent,
+    enabled:
+      (sentinelOneManualHostActionsEnabled && isSentinelOneAgent) ||
+      (crowdstrikeManualHostActionsEnabled && isSentinelOneAgent),
```

Comment:
```suggestion
      (crowdstrikeManualHostActionsEnabled && isCrowdStrikeAgent),
```

Reply:
This should also be enabled when it is `endpoint` agent and `agentStatusClientEnabled` is `true`.

Reply:
Good catch 👍  thanks!